zarf.yaml

```diff
@@ -15,6 +15,8 @@ components:
         namespace: podinfo
         kustomizations:
           - github.com/stefanprodan/podinfo//kustomize?ref=6.6.1
+        files:
+          - uds-package.yaml
```

Comment:
If this is also going to server as an example, do you want to put something not allowed in here as well and add an exemption?

Reply:
I don't think many people will use exemptions, but I did add an allow to show that off and since `echo` is a supported podinfo feature anyway.